corehq/apps/app_manager/dbaccessors.py

```diff
@@ -517,18 +536,32 @@ def get_case_types_from_apps(domain):
     save_to_case_updates = set()
     if VELLUM_SAVE_TO_CASE.enabled(domain):
         save_to_case_updates = _get_save_to_case_updates(domain)
-
-    case_types_agg = NestedAggregation('modules', 'modules').aggregation(
-        TermsAggregation('case_types', 'modules.case_type.exact'))
-    q = (AppES()
-         .domain(domain)
-         .is_build(False)
-         .size(0)
-         .aggregation(case_types_agg))
+    q = _get_case_types_from_apps_query(domain)
     case_types = set(q.run().aggregations.modules.case_types.keys)
     return (case_types.union(save_to_case_updates) - {''})
 
 
+def get_case_type_app_count(domain):
+    """
+    Gets the case types of modules in applications in the domain, returning
+    how many applications are associated with each case type.
+    :returns: A list of case types as the key and the number of associated apps as the value
+    """
+    q = _get_case_types_from_apps_query(domain)
+    case_types = q.run().aggregations.modules.case_types.counts_by_bucket()
+    return case_types
+
+
+def get_case_types_for_app_build(domain, app_id):
+    """
+    Gets the case types of modules for a specific application in the domain.
+    :returns: A set of case_types
+    """
+    q = _get_case_types_from_apps_query(domain, True, app_id)
```

Comment:
1. I'll add a named param for the `is_build` to make it clearer what the `True` value is for.
2. The reason I kept it in the private function was to keep all the logic for creating the query together. Since it's not being used anywhere else, it does make sense to rather pull it into `get_case_types_for_app_build()`. We can always move it back at a later stage if it ever gets needed by more functions.